sim/core/aura_helpers.go

```diff
@@ -369,12 +369,9 @@ func (parentAura *Aura) AttachSpellMod(spellModConfig SpellModConfig) *Aura {
 // Attaches a StatDependency to a parent Aura
 // Note: Only use when parent aura is used through RegisterAura() not GetOrRegisterAura. Otherwise this might apply multiple times.
 func (parentAura *Aura) AttachStatDependency(statDep *stats.StatDependency) *Aura {
-
 	parentAura.ApplyOnGain(func(_ *Aura, sim *Simulation) {
 		parentAura.Unit.EnableBuildPhaseStatDep(sim, statDep)
-	})
-
-	parentAura.ApplyOnExpire(func(_ *Aura, sim *Simulation) {
+	}).ApplyOnExpire(func(aura *Aura, sim *Simulation) {
```

Comment:
Pointless refactors that just add bloat 👩‍🍳

Reply:
😂